db/migrate/20191024132834_set_default_notification_settings.rb

```diff
@@ -0,0 +1,13 @@
+class SetDefaultNotificationSettings < ActiveRecord::Migration[5.1]
+  def up
+    User.all.each do |user|
+      user.preferences.digest_frequency = 'instant'
```

Comment:
we may need a `#save` here, or maybe we can remove the migration entirely (what are the chances of someone upgrading Dradis mid-project?)